vimeo-networking/src/main/java/com/vimeo/networking/model/Credit.java

```diff
@@ -54,11 +54,19 @@ public Video getVideo() {
         return mVideo;
     }
 
+    public void setVideo(@Nullable Video video) {
```

Comment:
Sounds good, I have already put field nullability on the agenda so this will fit in nicely